### DISCUSSION TITLE
See the Kanban note as markdown?

### DISCUSSION THREAD

**Discussion by tf13** [mgmeyers/obsidian-kanban#35] [Ideas & Feature Requests]
I'm sure I'm overlooking something. If I create a new Kanban board, it creates the underlying Markdown note. But is there some way I can see that note as a regular note in Obsidian? When I select it in Obsidian, I get the visual board. Normally that's what I want, but I'd also like to be able to see the markdown without having to open it in another app.

**Comment by danieltomasz** [comment]
Maybe you know this already, but there is a option "Open as markdown"

**Reply by tf13** [reply]
Yes -- I just found it when I checked back last night. I could have sworn it wasn't there when I posted, but maybe I overlooked it. Anyway, I'm impressed by how quickly development is going.
